mlflow/projects/__init__.py

```diff
@@ -204,7 +204,30 @@ def _run(uri, experiment_id, entry_point="main", version=None, parameters=None,
         )
         return submitted_run
 
-    supported_backends = ["local", "databricks", "kubernetes"]
+    # CRITEO
+    elif backend == "yarn":
+        from mlflow.projects import yarn
+
+        tracking.MlflowClient().set_tag(active_run.info.run_id, MLFLOW_PROJECT_BACKEND, "yarn")
+
+        submitted_run = yarn.run_yarn_job(
+            remote_run=active_run, uri=uri, entry_point_obj=entry_point_obj,
+            final_params=final_params, extra_params=extra_params,
+            experiment_id=experiment_id, pex_env=project.pex_env,
+            backend_config=backend_config
+        )
+
+        tracking.MlflowClient().set_tag(active_run.info.run_id,
+                                        yarn.MLFLOW_YARN_APPLICATION_ID,
+                                        submitted_run._skein_app_id)
```

Comment:
What is the point of this first tag ? If I understand correctly it is the same as the other one but with mlflow. before so it won't appear in the UI, couldn't we just remove it ?

Reply:
Yes.. I think this is a left over of the 1st version of the implementation on which I had no clue how the "mlflow.*" tags operated.

will remove